packages/app-page-builder-theme/src/components/Navigation.tsx

```diff
@@ -1,14 +1,6 @@
 import React from "react";
 import { Link } from "@webiny/react-router";
 
-function getLink({ url, title }) {
-    if (url.startsWith("/")) {
-        return <Link to={url}>{title}</Link>;
-    }
-
-    return <a href={url}>{title}</a>;
-}
-
```

Comment:
Good call! This is totally not needed since we handle this inside of the Link component 👍